pallets/capacity/src/lib.rs

```diff
@@ -302,31 +302,32 @@ pub mod pallet {
 		/// Staker attempted to stake to an invalid staking target.
 		InvalidTarget,
 		/// Capacity is not available for the given MSA.
-		InsufficientBalance,
+		InsufficientCapacityBalance,
 		/// Staker is attempting to stake an amount below the minimum amount.
-		InsufficientStakingAmount,
-		/// Staker is attempting to stake a zero amount.
+		StakingAmountBelowMinimum,
```

Comment:
Because of the rename below, I felt `InsufficientStakingAmount` would be too easy to confuse with the other one.